### ISSUE TITLE
[Test Failure][Accessibility] The cancel button of Choose NuGet Package Manager Format dialog truncated

### ISSUE THREAD

**Issue by v-luzh** [NuGet/Home#4607]
**NuGet Version:** VS.Redist.NuGet VS15.4.0.170213.1779
**VS Version:** D15Rel 26213.00 (select all languages and workloads)
**OS Version:** Windows 10 RS1_Release 14393.821.170127-1801

**Repro Steps:**
1.  Select RUS through ‘Tools -> Options -> International Settings’ and restart VS.
2.  Check **‘Allow format selection on first package install’** format in NuGet Package Manager setting dialog.
3.  Create a new C# Console Application via VS and click ‘Manage NuGet Packages…’.
4.  Select a package in ‘Browse’ tab and click Install button.
5.  Check the **Cancel** button in the opened **Choose NuGet Package Manager Format** dialog.

**Expected Result:**
Cancel button show correctly.

**Actual Result:**
Cancel button truncated as below screenshot.
![buttontruncated](https://cloud.githubusercontent.com/assets/20632625/22968239/97eb3fc6-f3a4-11e6-8e23-50c98f9e73ff.PNG)

**Note:**
It repro on ITA, PTB.

**Comment by v-luzh** [comment]
Fixed on D15PreRel 26626.02+NuGut Client 4.3.0.4226. Close this bug now, thanks.
